setup.py

```diff
@@ -0,0 +1,29 @@
+import setuptools
+import os
+
+REQUIRED = [
+    'pyhocon==0.3.54',
+    'requests==2.23.0',
+    'PyJWT==1.7.1'
+]
```

Comment:
Required versions in setup.py don't match required versions in requirements.txt and requirements_dev.txt. 
Another question, can we set less strict requirements like >= in setup.py, requirements.txt, and requirements_dev.txt files?